components/Layout.tsx

```diff
@@ -3,22 +3,24 @@ import { useRouter } from "next/router";
 import Footer from "./Footer";
 import Header from "./Header";
 
-export default function Layout({ children }) {
+export default function Layout({ children, page }) {
   const router = useRouter();
   const title = "Code Youngstown";
   const description =
     "Connecting software engineers, developers, and coders in the Youngstown, OH area. Join the ongoing chat via Slack and attend our meetups.";
+  const pageTitle = `${title} | ${page}`;
```

Comment:
Can we make it `${page} | ${title}` instead? And can we make it so that the home page has just the title `Code Youngsown`?